Sources/FoundationEssentials/Error/CocoaError+FilePath.swift

```diff
@@ -34,7 +37,10 @@ extension CocoaError.Code {
             case ENOENT: .fileNoSuchFile
             case EPERM, EACCES: .fileWriteNoPermission
             case ENAMETOOLONG: .fileWriteInvalidFileName
-            case EDQUOT, ENOSPC: .fileWriteOutOfSpace
+#if !os(Windows)
+            case EDQUOT: .fileWriteOutOfSpace
```

Comment:
Out of curiosity, what makes this case different than the others? I was surprised to see that this entire function wouldn't be `#if !os(Windows)`

Reply:
Windows is POSIX compliant, so there are cases where we want to return POSIX errors. However, `EDQUOT` is not defined on Windows.